pom.xml

```diff
@@ -8,7 +8,7 @@
         <version>22</version>
     </parent>
 
-    <groupId>io.takari.maven.plugins</groupId>
+    <groupId>com.facebook.presto</groupId>
```

Comment:
We need this so we can release a new version to Maven central. This project was originally written by @jvanzyl (who owns Takari) and I don't want to bother him every time we need to release :)